primitives/src/asset.rs

```diff
@@ -64,3 +67,23 @@ enum_with_aux_fns! {
         Vez = "VEZ",
```

Comment:
Since the clippy restriction is already lifted, maybe make all currencies upper case?